thingsboard_gateway/connectors/opcua/opcua_connector.py

```diff
@@ -134,6 +135,13 @@ async def __cancel_all_tasks(self):
         for task in asyncio.all_tasks(self.__loop):
             task.cancel()
 
+    async def __disconnect(self):
+        try:
+            await self.__client.disconnect()
+            self.__log.info('%s has been disconnected from OPC-UA Server.', self.get_name())
+        except:
+             self.__log.warning(f'{self.get_name()} could not be disconnected from OPC-UA Server.') 
```

Comment:
It is not the best practice to use f'...', so I'd like to suggest using the following code here:
```python
self.__log.error('%s could not be disconnected from OPC-UA Server: %s', self.name, e)
```